nparse_py.spec

```diff
@@ -9,7 +9,7 @@ a = Analysis(['nparse.py'],
              datas=[],
              hiddenimports=[],
              hookspath=[],
-             runtime_hooks=[],
+             runtime_hooks=['set_qt_conf.py'],
```

Comment:
QtWebEngine packaging is ... wonky. This was the only way I could figure out how to make it set up the paths right on deployment when using `--onefile` build mode.